github/Repository.py

```diff
@@ -2089,6 +2089,22 @@ def create_fork(self, organization=github.GithubObject.NotSet):
         )
         return Repository(self._requester, headers, data, completed=True)
 
+    def transfer(self, new_owner, team_ids=[]):
+        """
+        :calls: `POST /repos/:owner/:repo/transfer <https://developer.github.com/v3/repos/#transfer-a-repository>`_
+        :param new_owner: string or "none" or "*". Required.
+        :param team_ids: list. Optional. Works only under organizations.
+        :rtype: :class:`github.Repository.Repository`
+        """
+        post_parameters = {}
+        if new_owner is not None:
```

Comment:
We require this argument, so this check is pointless. Please add asserts for both arguments.

Reply:
not sure how to proceed with unit test for "transfer" as it requires GitHub permissions to 2 organizations etc. so not-so-trivial to have set up. Any pointers?